.bazelrc

```diff
@@ -1,11 +1,10 @@
 common --enable_bzlmod
-build --enable_platform_specific_config
```

Comment:
reason for this change? it's used down below. see `build:linux` and `build:macos`

Reply:
Good catch! Removed it.